scalability_and_performance/ztp-deploying-disconnected.adoc

```diff
@@ -37,7 +44,8 @@ include::modules/installation-about-mirror-registry.adoc[leveloffset=+2]
 [role="_additional-resources"]
 .Additional resources
 
-* For information on viewing the CRI-O logs to view the image source, see xref:../installing/validating-an-installation.html#viewing-the-image-pull-source_validating-an-installation[Viewing the image pull source].
+* For information on viewing the CRI-O logs to view the image source,
+see xref:../installing/validating-an-installation.html#viewing-the-image-pull-source_validating-an-installation[Viewing the image pull source].
 
```

Comment:
I really want to do overhaul this section as you suggest, but I think it will make more sense to handle this with the ZTP docs refactor epic. Also we're running out of time for this release. I'm going to leave this as is for now.

Reply:
@imiller0 I am going to talk to Yang to see if she can support this from a QE perspective. If she agrees I will make this change.

Reply:
Verified. This is implemented.